pkg/rke/templates/templates.go

```diff
@@ -271,7 +272,8 @@ func LoadK8sVersionedTemplates() map[string]map[string]string {
 			">=1.23.16-rancher1-1 <1.24.0-rancher1-1":  nginxIngressv151Rancher2,
 			">=1.24.2-rancher1-1 <1.24.10-rancher1-1":  nginxIngressv120Rancher2,
 			">=1.24.10-rancher1-1 <1.26.0-rancher1-1":  nginxIngressv151Rancher2,
-			">=1.26.0-rancher1-1":                      nginxIngressv170Rancher1,
+			">=1.26.0-rancher1-1 <1.27.2-rancher1-1":   nginxIngressv170Rancher1,
+			">=1.27.2-rancher1-1":                      nginxIngressv181Rancher1,
```

Comment:
Question: why Nginx v1.8.1 is used for v1.27.2 and above, not v1.27.0 and above?

Reply:
We were introducing 1.27.2 version back then for 1.27 support so I kept it as 1.27.2. will change it to 1.27.0 and update the PR.

Reply:
Update the PR